server/controllers/favouritesControllers.js

```diff
@@ -0,0 +1,109 @@
+import db from '../models/index';
+
+
+/**
+ *
+ *
+ * @export
+ * @class Favourite
+ */
+export default class Favourite {
+  /**
+   *
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} with recipe added as favourite
+   * @memberof Favourite
+   */
+  static addFavourite(req, res) {
+    if (typeof parseInt(req.params.recipeId, 10) !== 'number') {
```

Comment:
### Critical
If you are type-checking for a number value, always favour `isNaN` over `typeof` because typeof will return `number` after type casting an actual string value but `isNaN` will return a boolean that asserts that the value is  really a number. See example below:
```
let a = '5';
let x = 'femi';

// assert that each of them is actually a number
isNaN(parseInt(a, 10)); // false
isNaN(parseInt(x, 10)); // true
```